src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
```

Comment:
We need UX people.
Maybe it makes sense that the screen reader don't speak when there is no security image. Instead of speaking "no security image".